README.md

```diff
@@ -321,6 +321,31 @@ and add them to the media library if their MIME types are supported. If the
 file/folder in _path_ does not exist/is not readable or is not provided then an
 error will be returned and the corresponding log message would be written into logs.
 
+## Internal Audio & Video Recording
+
+REQUIRED STEPS to activate recording:
+
+```bash
+adb shell pm grant io.appium.settings android.permission.RECORD_AUDIO
+adb shell pm grant io.appium.settings android.permission.WRITE_EXTERNAL_STORAGE
+adb shell appops set io.appium.settings PROJECT_MEDIA allow
+```
+
+Start Recording:
+```bash
+adb shell am start -n "io.appium.settings/io.appium.settings.Settings" -a io.appium.settings.recording.ACTION_START --es "recording_filename" "abc.mp4"
```

Comment:
Please add descriptions for all supported arguments

Reply:
done https://github.com/sirmordred/io.appium.settings/commit/a49f98efb54ac6cca1e6cc3e089fcb596cb5b2d4